src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -275,34 +275,52 @@ void DynamoView_MouseLeftButtonDown(object sender, MouseButtonEventArgs e)
 
         private void DynamoViewModel_RequestEnableShortcutBarItems(bool enable)
         {
-            saveThisButton.IsEnabled = enable;
-            saveButton.IsEnabled = enable;
-            exportMenu.IsEnabled = enable;
-            
-            shortcutBar.IsNewButtonEnabled = enable;
-            shortcutBar.IsOpenButtonEnabled = enable;
-            shortcutBar.IsSaveButtonEnabled = enable;
-            shortcutBar.IsLoginMenuEnabled = enable;
-            shortcutBar.IsExportMenuEnabled  = enable;
-            shortcutBar.IsNotificationCenterEnabled = enable;
-
-            if(dynamoViewModel.ShowStartPage)
+            if (!(saveThisButton is null))
```

Comment:
wrapped these in null checks, as this method get called during workspace shutdown, but if just the splash screen is visible these items are null and the splash screen is never closed.